src/lib/swapper/swappers/ThorchainSwapper/utils/parseThorBuyTxHash.ts

```diff
@@ -13,7 +13,7 @@ export const parseThorBuyTxHash = (
   // swaps into rune aren't double swaps so don't have a second tx (buy tx)
   if (!isDoubleSwap) return sellTxId
 
-  const isEvmCoinAsset = !THORCHAIN_EVM_CHAINS.some(
+  const isEvmCoinAsset = THORCHAIN_EVM_CHAINS.some(
```

Comment:
note bug fix, it was pre-pending a `0x` where it should not have (i.e UTXO tx hashes) due to inverted logic